src/main/utils/config.ts

```diff
@@ -95,14 +95,20 @@ export async function readValueForType(raw: string, type: ObjectType): Promise<a
                     return Promise.resolve(parseFloat(raw))
 
                 case 'boolean':
-                    return Promise.resolve(raw === 'true')
+                    if (raw === 'true') {
+                        return Promise.resolve(true)
+                    } else if (raw === 'false') {
+                        return Promise.resolve(false)
+                    } else {
+                        throw new DynamicConfigInvalidType(key, type)
+                    }
```

Comment:
Nice, I like how you tightened up the string => boolean parsing. The old behavior was pretty sketchy.